scene/gui/tabs.cpp

```diff
@@ -419,13 +418,16 @@ int Tabs::get_tab_count() const {
 
 void Tabs::set_current_tab(int p_current) {
 
+	if (current == p_current) return;
```

Comment:
I think Akien wants to avoid using one-line ifs.
Quoting Akien from 9702:
>Yeah I also prefer that we avoid one-line ifs, they make it harder to distinguish the condition from the actual if block.